pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -1,11 +1,11 @@
-@import 'color', 'font', 'typography', 'breakpoints', 'barlow-semi-condensed-font';
+@import 'font', 'typography', 'breakpoints';
```

Comment:
Keeping `'font'` and `'typography'` since some areas still use mixins from these imports.